docs/getting-started.md

```diff
@@ -1,20 +1,19 @@
 Hi, are you new to [Zulip](https://github.com/zulip/zulip)? If so, we recommend trying out our [web-client](https://chat.zulip.org) first to understand the concept of [streams/topics/PMs](https://zulipchat.com/help/about-streams-and-topics) in the world of Zulip. Just sending a message or two in [#test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream should help you get started.
 
-Now let's help you get started with using zulip terminal. First, go ahead and complete the [installation](https://github.com/zulip/zulip-terminal#installation--running). If you encountered any issues, we have common issues and their solutions listed [here](https://github.com/zulip/zulip-terminal#troubleshooting-common-issues).
+Now let's help you get started with using zulip terminal. First, go ahead and complete the [installation](https://github.com/zulip/zulip-terminal#installation). If you encountered any issues, we have common issues and their solutions listed [here](https://github.com/zulip/zulip-terminal#troubleshooting-common-issues).
 
 If you encountered any issues above or have any queries, feel free to ask for help at [#zulip-terminal](https://chat.zulip.org/#narrow/stream/206-zulip-terminal) or support@zulipchat.com.
 
-## Tutorial
-I would like you to try out a few of our keyboard shortcuts and send a couple message to help you get familiar with zulip terminal.
+Once you get are done with the installation steps, refer the [runner for the first time](https://github.com/zulip/zulip-terminal#running-for-the-first-time) section and the [configuration](https://github.com/zulip/zulip-terminal#configuration) section of the README.
+
 
-#### What do you see?
 ![An image of working Zulip Terminal](https://camo.githubusercontent.com/d4efe459c212e64ae7c841e5dda288a14837bf1e/68747470733a2f2f64726976652e676f6f676c652e636f6d2f75633f6578706f72743d766965772669643d31746163325053535f47625177587a34476e7a66636f5f68696e5776464f327a61)
+
 Zulip Terminal is divided into three columns/divisions.
 **The left column** shows list of all the streams you are subscribed to and some buttons for viewing different types of messages.
-**The middle column** displays all the messages in the current narrow. 
+**The middle column** displays all the messages in the current narrow.
 **The right column** shows all the active(green)/idle(yellow)/offline(white) users.
 
-Now that you know what you are looking at, how about checking out some of our features using [keyboard shortcuts](https://github.com/zulip/zulip-terminal#hot-keys)... 
```

Comment:
It would be cleaner to tidy the end characters in a separate commit across all the docs?